File: Ejers 100 - 199/p191.java
import java.util.Scanner;

public class p191 {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int c = s.nextInt();

        while (c-- != 0) {
            int resultado = 0;
            int contador = s.nextInt();
            int litros = s.nextInt();
            resultado += litros;
            int diferencia = s.nextInt();

            for (int i = 1; i < contador; i++) {
                litros -= diferencia;
                resultado = resultado + litros;
            }
            System.out.println(resultado);
        }
    }
}

File: C++/p191.cpp
/* 
Ejercicio de AceptaElReto - P191 (Los problemas de ser rico)
Enlace: https://aceptaelreto.com/problem/statement.php?id=191
*/

#include <iostream>
#include <string>

using namespace std;

int main()
{
    int casos;
    cin >> casos;
    
    while (casos-- != 0) {
        int result, contador, litros, diferencia;
        
        cin >> contador >> litros >> diferencia;
        
        result = litros;
        
        for (int i = 1; i < contador; i++) {
            litros -= diferencia;
            result += litros;
        }
        cout << result << endl;
    }
    return 0;
}
